js/main.js

```diff
@@ -12,13 +12,48 @@ var $ul = document.querySelector('ul');
 var $titleInput = document.getElementById('title-input');
 var $notesInput = document.getElementById('notes-input');
 var $headerText = document.getElementById('header-text');
+var $delete = document.getElementById('delete-entry');
+var $modal = document.querySelector('.modal-overlay');
+var $cancel = document.querySelector('.cancel');
+var $confirm = document.querySelector('.confirm');
 
 $photoInput.addEventListener('input', handleInput);
 $createForm.addEventListener('submit', handleSubmit);
 window.addEventListener('DOMContentLoaded', handleLoad);
 $entriesLink.addEventListener('click', dataView);
 $newLink.addEventListener('click', dataView);
 $ul.addEventListener('click', editEntry);
+$delete.addEventListener('click', handleDelete);
+$cancel.addEventListener('click', handleCancel);
+$confirm.addEventListener('click', handleConfirmDOM);
+$confirm.addEventListener('click', handleConfirmData);
```

Comment:
You should just make this into one event listener since both pieces of work are related to the same job.